arcgis-java-toolkit/src/main/java/com/esri/arcgisruntime/toolkit/skins/UtilityNetworkTraceStartingPointView.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2022 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.esri.arcgisruntime.toolkit.skins;
+
+import java.awt.*;
+import java.util.concurrent.TimeUnit;
+
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.toolkit.UtilityNetworkTraceStartingPoint;
+import com.esri.arcgisruntime.utilitynetworks.UtilityTerminal;
+import javafx.beans.binding.Bindings;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.Label;
+import javafx.scene.control.Slider;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Priority;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.util.StringConverter;
+
+/**
+ * A custom BorderPane for a starting point displayed in a {@link UtilityNetworkTraceSkin}.
+ *
+ * <p>
+ * Has custom style class applied if required for customization:
+ * utility-network-trace-starting-point-view
+ *
+ * @since 100.15.0
+ */
+public class UtilityNetworkTraceStartingPointView extends BorderPane {
+
+  /**
+   * Creates a UtilityNetworkTraceStartingPointView.
+   *
+   * @param skin the UtilityNetworkTraceSkin skin where the starting point is displayed
+   * @param startingPoint the UtilityNetworkTraceStartingPoint model for the starting point
+   * @since 100.15.0
+   */
+  protected UtilityNetworkTraceStartingPointView(
+    UtilityNetworkTraceSkin skin, UtilityNetworkTraceStartingPoint startingPoint) {
+    double height = skin.getStartingPointListCellHeight() - 10;
+    this.setPrefHeight(height);
+    this.setMinHeight(height);
+    this.setMaxHeight(height + 15);
+    this.setPadding(new Insets(5, 2, 5, 2));
+    this.getStyleClass().add("utility-network-trace-starting-point-view");
+
+    // Left of the borderpane is a thumbnail image of the feature symbol.
+    // Setup and configure the feature symbol image
+    var symbolVBox = new VBox();
+    symbolVBox.setAlignment(Pos.CENTER);
+    symbolVBox.setMinWidth(40);
+    var featureSymbolImageView = new ImageView();
+    symbolVBox.getChildren().add(featureSymbolImageView);
+    try {
+      featureSymbolImageView.setImage(
+        startingPoint.getFeatureSymbol().createSwatchAsync(Color.TRANSLUCENT, 1f).get(30, TimeUnit.SECONDS));
```

Comment:
30s seems a long time to potentially block. Assuming `startingPoint.getFeatureSymbol()` is just an in memory symbol 1 or 2 seconds should be enough... say 5 to be safe?